diff -uNr a/debian/libre2-9.symbols b/debian/libre2-9.symbols
--- a/debian/libre2-9.symbols	2022-06-02 02:59:28.000000000 +0800
+++ b/debian/libre2-9.symbols	2023-06-03 02:59:51.000000000 +0800
@@ -1,7 +1,7 @@
 libre2.so.9 libre2-9 #MINVER#
 * Build-Depends-Package: libre2-dev
-(arch= alpha  amd64  arm64  ia64  mips64  mips64el  ppc64  ppc64el  riscv64  s390x  sparc64  netbsd-alpha  kfreebsd-amd64  s390  avr32)#include "libre2.symbols.long_size_t"
-(arch=!alpha !amd64 !arm64 !ia64 !mips64 !mips64el !ppc64 !ppc64el !riscv64 !s390x !sparc64 !netbsd-alpha !kfreebsd-amd64 !s390 !avr32)#include "libre2.symbols.int_size_t"
+(arch= alpha  amd64  arm64  ia64 loong64 mips64  mips64el  ppc64  ppc64el  riscv64  s390x  sparc64  netbsd-alpha  kfreebsd-amd64  s390  avr32)#include "libre2.symbols.long_size_t"
+(arch=!alpha !amd64 !arm64 !ia64 !loong64 !mips64 !mips64el !ppc64 !ppc64el !riscv64 !s390x !sparc64 !netbsd-alpha !kfreebsd-amd64 !s390 !avr32)#include "libre2.symbols.int_size_t"
  _ZN3re211FilteredRE214PrintPrefilterEi@Base 20131024+dfsg
  _ZN3re211FilteredRE219RegexpsGivenStringsERKSt6vectorIiSaIiEEPS3_@Base 20131024+dfsg
  _ZN3re211FilteredRE23AddERKNS_11StringPieceERKNS_3RE27OptionsEPi@Base 20131024+dfsg
