diff -uNr a/debian/control b/debian/control
--- a/debian/control	2022-12-28 09:08:15.000000000 +0800
+++ b/debian/control	2023-06-22 04:08:31.000000000 +0800
@@ -22,7 +22,7 @@
 Rules-Requires-Root: no
 
 Package: rakudo
-Architecture: amd64 arm64 armel armhf i386 kfreebsd-amd64 kfreebsd-i386 mips mips64el mipsel powerpc ppc64 ppc64el riscv64 s390x x32
+Architecture: amd64 arm64 armel armhf i386 kfreebsd-amd64 kfreebsd-i386 loong64 mips mips64el mipsel powerpc ppc64 ppc64el riscv64 s390x x32
 Depends: ${misc:Depends},
          ${moarvm:Depends},
          ${nqp:Depends},
