diff -uNr a/debian/control b/debian/control
--- a/debian/control	2023-02-11 02:27:14.000000000 +0800
+++ b/debian/control	2023-06-22 00:26:18.000000000 +0800
@@ -15,7 +15,7 @@
                gtk-doc-tools,
                libadwaita-1-dev (>= 1.0.1),
                libappstream-dev (>= 0.14.0),
-               libflatpak-dev (>= 1.14.0) [amd64 arm64 armel armhf i386 mipsel mips64el ppc64el s390x hppa powerpc ppc64 riscv64 x32],
+               libflatpak-dev (>= 1.14.0) [amd64 arm64 armel armhf i386 loong64 mipsel mips64el ppc64el s390x hppa powerpc ppc64 riscv64 x32],
                libfwupd-dev (>= 1.5.6) [linux-any],
                libglib2.0-dev (>= 2.70.0),
                libglib-testing-0-dev,
@@ -23,10 +23,10 @@
                libgudev-1.0-dev [linux-any],
                libjson-glib-dev (>= 1.6.0),
                libostree-dev (>= 2022.5-2~) [linux-any],
-               libmalcontent-0-dev [amd64 arm64 armel armhf i386 mipsel mips64el ppc64el s390x hppa powerpc ppc64 riscv64 x32],
+               libmalcontent-0-dev [amd64 arm64 armel armhf i386 loong64 mipsel mips64el ppc64el s390x hppa powerpc ppc64 riscv64 x32],
                libpackagekit-glib2-dev (>= 1.1.11),
                libpolkit-gobject-1-dev,
-               libsnapd-glib-dev (>= 1.63) [amd64 arm64 armel armhf i386 ppc64el s390x],
+               libsnapd-glib-dev (>= 1.63) [amd64 arm64 armel armhf i386 loong64 ppc64el s390x],
                libsoup-3.0-dev,
                libxml2-utils,
                libxmlb-dev (>= 0.1.7),
@@ -61,7 +61,7 @@
 Conflicts: sessioninstaller
 Recommends: fwupd [linux-any], ${plugin:Recommends}
 Suggests: apt-config-icons-hidpi,
-          gnome-software-plugin-flatpak [amd64 arm64 armel armhf i386 mipsel mips64el ppc64el s390x hppa powerpc ppc64 riscv64 x32],
+          gnome-software-plugin-flatpak [amd64 arm64 armel armhf i386 loong64 mipsel mips64el ppc64el s390x hppa powerpc ppc64 riscv64 x32],
           ${plugin:Suggests}
 Description: Software Center for GNOME
  Software lets you install and update applications and system extensions.
@@ -88,7 +88,7 @@
  This package contains the architecture-independent files.
 
 Package: gnome-software-plugin-flatpak
-Architecture: amd64 arm64 armel armhf i386 mipsel mips64el ppc64el s390x hppa powerpc ppc64 riscv64 x32
+Architecture: amd64 arm64 armel armhf i386 loong64 mipsel mips64el ppc64el s390x hppa powerpc ppc64 riscv64 x32
 Depends: flatpak (>= 1.14.0),
          gnome-software (= ${binary:Version}),
          ${misc:Depends},
@@ -105,7 +105,7 @@
  This package contains the Flatpak plugin.
 
 Package: gnome-software-plugin-snap
-Architecture: amd64 arm64 armel armhf i386 ppc64el s390x
+Architecture: amd64 arm64 armel armhf i386 loong64 ppc64el s390x
 Depends: gnome-software (= ${binary:Version}),
          snapd [amd64 arm64 armel armhf i386 ppc64el],
          ${misc:Depends},
diff -uNr a/debian/control.in b/debian/control.in
--- a/debian/control.in	2023-02-11 02:27:14.000000000 +0800
+++ b/debian/control.in	2023-06-22 00:26:18.000000000 +0800
@@ -11,7 +11,7 @@
                gtk-doc-tools,
                libadwaita-1-dev (>= 1.0.1),
                libappstream-dev (>= 0.14.0),
-               libflatpak-dev (>= 1.14.0) [amd64 arm64 armel armhf i386 mipsel mips64el ppc64el s390x hppa powerpc ppc64 riscv64 x32],
+               libflatpak-dev (>= 1.14.0) [amd64 arm64 armel armhf i386 loong64 mipsel mips64el ppc64el s390x hppa powerpc ppc64 riscv64 x32],
                libfwupd-dev (>= 1.5.6) [linux-any],
                libglib2.0-dev (>= 2.70.0),
                libglib-testing-0-dev,
@@ -19,10 +19,10 @@
                libgudev-1.0-dev [linux-any],
                libjson-glib-dev (>= 1.6.0),
                libostree-dev (>= 2022.5-2~) [linux-any],
-               libmalcontent-0-dev [amd64 arm64 armel armhf i386 mipsel mips64el ppc64el s390x hppa powerpc ppc64 riscv64 x32],
+               libmalcontent-0-dev [amd64 arm64 armel armhf i386 loong64 mipsel mips64el ppc64el s390x hppa powerpc ppc64 riscv64 x32],
                libpackagekit-glib2-dev (>= 1.1.11),
                libpolkit-gobject-1-dev,
-               libsnapd-glib-dev (>= 1.63) [amd64 arm64 armel armhf i386 ppc64el s390x],
+               libsnapd-glib-dev (>= 1.63) [amd64 arm64 armel armhf i386 loong64 ppc64el s390x],
                libsoup-3.0-dev,
                libxml2-utils,
                libxmlb-dev (>= 0.1.7),
@@ -57,7 +57,7 @@
 Conflicts: sessioninstaller
 Recommends: fwupd [linux-any], ${plugin:Recommends}
 Suggests: apt-config-icons-hidpi,
-          gnome-software-plugin-flatpak [amd64 arm64 armel armhf i386 mipsel mips64el ppc64el s390x hppa powerpc ppc64 riscv64 x32],
+          gnome-software-plugin-flatpak [amd64 arm64 armel armhf i386 loong64 mipsel mips64el ppc64el s390x hppa powerpc ppc64 riscv64 x32],
           ${plugin:Suggests}
 Description: Software Center for GNOME
  Software lets you install and update applications and system extensions.
@@ -84,7 +84,7 @@
  This package contains the architecture-independent files.
 
 Package: gnome-software-plugin-flatpak
-Architecture: amd64 arm64 armel armhf i386 mipsel mips64el ppc64el s390x hppa powerpc ppc64 riscv64 x32
+Architecture: amd64 arm64 armel armhf i386 loong64 mipsel mips64el ppc64el s390x hppa powerpc ppc64 riscv64 x32
 Depends: flatpak (>= 1.14.0),
          gnome-software (= ${binary:Version}),
          ${misc:Depends},
@@ -101,7 +101,7 @@
  This package contains the Flatpak plugin.
 
 Package: gnome-software-plugin-snap
-Architecture: amd64 arm64 armel armhf i386 ppc64el s390x
+Architecture: amd64 arm64 armel armhf i386 loong64 ppc64el s390x
 Depends: gnome-software (= ${binary:Version}),
          snapd [amd64 arm64 armel armhf i386 ppc64el],
          ${misc:Depends},
diff -uNr a/debian/rules b/debian/rules
--- a/debian/rules	2023-02-11 02:27:14.000000000 +0800
+++ b/debian/rules	2023-06-22 00:26:18.000000000 +0800
@@ -22,7 +22,7 @@
 
 ifeq ($(DEB_HOST_ARCH_OS), linux)
 	# Enable Flatpak support on supported architectures
-	ifneq (,$(filter $(DEB_HOST_ARCH), amd64 arm64 armel armhf i386 mipsel mips64el ppc64el s390x hppa powerpc ppc64 riscv64 x32))
+	ifneq (,$(filter $(DEB_HOST_ARCH), amd64 arm64 armel armhf i386 loong64 mipsel mips64el ppc64el s390x hppa powerpc ppc64 riscv64 x32))
 		GS_CONFIGURE_FLAGS += -Dflatpak=true -Dmalcontent=true
 	endif
 
@@ -33,7 +33,7 @@
 	GS_CONFIGURE_FLAGS += -Dfwupd=true
 
 	# Enable snap support on supported architectures
-	ifneq (,$(filter $(DEB_HOST_ARCH), amd64 arm64 armel armhf i386 ppc64el s390x))
+	ifneq (,$(filter $(DEB_HOST_ARCH), amd64 arm64 armel armhf i386 loong64 ppc64el s390x))
 		GS_CONFIGURE_FLAGS += -Dsnap=true
 	endif
 endif
