diff -uNr a/debian/control b/debian/control
--- a/debian/control	2023-01-30 01:32:20.000000000 +0800
+++ b/debian/control	2023-06-21 23:10:13.000000000 +0800
@@ -15,7 +15,7 @@
                dh-sequence-gnome,
                docbook-xml,
                gettext (>= 0.19.8),
-               gjs [amd64 armel arm64 armhf i386 mips64el mipsel ppc64el s390x riscv64],
+               gjs [amd64 armel arm64 armhf i386 loong64 mips64el mipsel ppc64el s390x riscv64],
                gnome-pkg-tools (>= 0.16.3),
                gnome-settings-daemon-dev (>= 3.24.0),
                gobject-introspection (>= 0.10.7-1~),
@@ -49,7 +49,7 @@
 Homepage: https://wiki.gnome.org/Projects/GDM/
 
 Package: gdm3
-Architecture: amd64 arm64 armel armhf i386 mips64el mipsel ppc64el s390x riscv64
+Architecture: amd64 arm64 armel armhf i386 loong64 mips64el mipsel ppc64el s390x riscv64
 Pre-Depends: ${misc:Pre-Depends}
 Depends: accountsservice (>= 0.6.35),
          adduser,
diff -uNr a/debian/control.in b/debian/control.in
--- a/debian/control.in	2023-01-30 01:32:20.000000000 +0800
+++ b/debian/control.in	2023-06-21 23:10:11.000000000 +0800
@@ -11,7 +11,7 @@
                dh-sequence-gnome,
                docbook-xml,
                gettext (>= 0.19.8),
-               gjs [amd64 armel arm64 armhf i386 mips64el mipsel ppc64el s390x riscv64],
+               gjs [amd64 armel arm64 armhf i386 loong64 mips64el mipsel ppc64el s390x riscv64],
                gnome-pkg-tools (>= 0.16.3),
                gnome-settings-daemon-dev (>= 3.24.0),
                gobject-introspection (>= 0.10.7-1~),
@@ -45,7 +45,7 @@
 Homepage: https://wiki.gnome.org/Projects/GDM/
 
 Package: gdm3
-Architecture: amd64 arm64 armel armhf i386 mips64el mipsel ppc64el s390x riscv64
+Architecture: amd64 arm64 armel armhf i386 loong64 mips64el mipsel ppc64el s390x riscv64
 Pre-Depends: ${misc:Pre-Depends}
 Depends: accountsservice (>= 0.6.35),
          adduser,
